modules/tappxBidAdapter.js

```diff
@@ -253,6 +264,25 @@ function buildOneRequest(validBidRequests, bidderRequest) {
   if (config.getConfig('coppa') === true) {
     regs.coppa = config.getConfig('coppa') === true ? 1 : 0;
   }
+
+  // Universal ID
+  const eidsArr = utils.deepAccess(validBidRequests, 'userIdAsEids');
+
+  let eids = [];
+  eidsArr.forEach(eidsElement => {
+    if (eidsElement.source && eidsElement.uids[0].id) {
+      eids.push({
+        source: eidsElement.source,
+        userId: eidsElement.uids[0].id
+      });
+    }
+  });
```

Comment:
what is the purpose of this block? the `eids` array is not used anywhere else (on line 283 you are setting `user.ext.eids = eidsArr`).

I would suggest passing the entire `eidsArr` as you have the code below, since some ID providers include extensions to their ID that could be useful to you or your downstream DSP partners, so passing them as defined in the `userIdAsEids` array is recommended).

Reply:
Initially we thought of adding the `eids` array only with the **source** and **uids** but we've not implemented it like that. We left the code by mistake. As you said, it has been removed and we directly use eidsArr. Commit: 37d9f96

Reply:
sounds good!